client/src/common/model/actions.js

```diff
@@ -0,0 +1,56 @@
+import _upperFirst from 'lodash/upperFirst';
+
+import socket from 'sockets';
+import { ListActionTypes } from 'modules/list/model/actionTypes';
+import { CohortActionTypes } from 'modules/cohort/model/actionTypes';
+import { Routes } from 'common/constants/enums';
+
+const clearListMetaDataSuccess = () => ({
+  type: ListActionTypes.CLEAR_META_DATA_SUCCESS
+});
+
+const clearCohortMetaDataSuccess = () => ({
+  type: CohortActionTypes.CLEAR_META_DATA_SUCCESS
+});
+
+export const enterView = (route, userId) => dispatch => {
+  const view = _upperFirst(route);
+
+  socket.emit(`enter${view}View`, userId);
```

Comment:
Please try not to use computed ids. Rather, send the view as a parameter of generalized action:
```socket.emit('enterView', { userId, view });```